File: app/src/main/java/com/learning/ruslan/Suffix.java
package com.learning.ruslan;

public class Suffix extends Task {
    private String alternative;

    public Suffix(int id, String word, int position, String alternative_letter) {
        super(id, word, position);
        setAlternative(alternative_letter);
    }

    public Suffix(String word, int position, String alternative_letter) {
        this(0, word, position, alternative_letter);
    }

//    public void set(String word, int position, String alternative_letter) {
//        setWord(word);
//        setPosition(position);
//        setAlternative(alternative_letter);
//    }



    public String getAlternative() {
        return alternative;
    }

    public void setAlternative(String alternativeLetter) {
        this.alternative = alternativeLetter;
    }
}

File: app/src/main/java/com/learning/ruslan/task/Suffix.kt
package com.learning.ruslan.task

import androidx.room.Entity
import androidx.room.Ignore

@Entity(tableName = "SuffixTable")
class Suffix(id: Int, word: String, position: Int, var alternative: String) :
    Task(word, position, id) {

    @Ignore
    constructor(word: String, position: Int, alternative_letter: String) :
            this(
                id = 0,
                word = word,
                position = position,
                alternative = alternative_letter
            )
}